lib/install.go

```diff
@@ -363,28 +342,54 @@ func InstallOption(listAll, dryRun bool, customBinaryPath, installPath string, m
 	InstallProductOption(product, listAll, dryRun, customBinaryPath, installPath, mirrorURL)
 }
 
+type VersionSelector struct {
+	Version string
+	Label   string
+}
+
 // InstallOption displays & installs tf version
 /* listAll = true - all versions including beta and rc will be displayed */
 /* listAll = false - only official stable release are displayed */
 func InstallProductOption(product Product, listAll, dryRun bool, customBinaryPath, installPath string, mirrorURL string) {
-	tflist, _ := getTFList(mirrorURL, listAll)                   // Get list of versions
-	recentVersions, _ := getRecentVersions(product, installPath) // Get recent versions from RECENT file
-	tflist = append(recentVersions, tflist...)                   // Append recent versions to the top of the list
-	tflist = removeDuplicateVersions(tflist)                     // Remove duplicate version
+	var selectVersions []VersionSelector
+
+	// Add recent versions
+	for _, version := range getRecentVersions(product, installPath) {
+		selectVersions = append(selectVersions, VersionSelector{
+			Version: version,
+			Label:   version + " *recent",
+		})
+	}
+
+	// Add all versions
+	tfList, _ := getTFList(mirrorURL, listAll)
+	for _, version := range tfList {
+		selectVersions = append(selectVersions, VersionSelector{
+			Version: version,
+			Label:   version,
+		})
+	}
 
-	if len(tflist) == 0 {
+	if len(selectVersions) == 0 {
 		logger.Fatalf("Terraform version list is empty: %s", mirrorURL)
 		os.Exit(1)
 	}
 
 	/* prompt user to select version of terraform */
 	prompt := promptui.Select{
 		Label: "Select Terraform version",
```

Comment:
Add `TODO` note to replace `Terraform` with name of Product when PR for OpenTofu support is approved and merged? 🤔

Reply:
:+1: Because of the changes in the OpenTofu branch, this a branch from that branch, so all of this is immediately available to implement.
As part of the other branch, I'll take a wider look at how many places "Terraform" is still being used in messages etc

Reply:
> this a branch from that branch, so all of this is immediately available to implement.

ouch, I had a gut feeling akin to deja vu 🤣 

> As part of the other branch, I'll take a wider look at how many places "Terraform" is still being used in logs etc

Appreciate that 🙏🏻

Reply:
Fixed :)